src/module/apikeys/index.ts

```diff
@@ -27,3 +27,7 @@ export const PermissionModel = "permission.venturemark.co/model";
 export const TimelineLastUpdate = "timeline.venturemark.co/last-update";
 
 export const UpdateFormat = "update.venturemark.co/format";
+
+export const UserPrepopulate = "user.venturemark.co/prepopulate";
+
+export const UserSurveyResponse = "user.venturemark.co/surveyResponse";
```

Comment:
these two keys are placeholder, will change later @tfussell

Reply:
are you ok with this change? @tfussell

Reply:
These are fine.